### PR TITLE
add name

### PR DESCRIPTION
**Pull request by nemesis04**
add name

### CODE REVIEW

File: installation.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-## **Installation**
+## **Installation guide**
+ไอซ์
+ฮ้วง
```
Comment (nemesis04) [inline installation.md:3]: wrong word?

File: installation.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+เอิก
+ไกด์
+นนท์
```
Comment (nemesis04) [inline installation.md:6]: same

**Review by jsice — APPROVED** [review]

File: installation.md (hunk 2 above)
Comment (NonXtreme) [inline installation.md:6]: wrong name?

**Review by huangsri — APPROVED** [review]

File: installation.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
-## **Installation**
+## **Installation guide**
+ไอซ์
+ฮ้วง
+เอิก
```
Comment (aerkmukdahan) [inline installation.md:4]: awesome

**Review by NonXtreme — APPROVED** [review]

**Pull request merged**